toolkits/global/packages/global-button/scss/30-mixins/button.scss

```diff
@@ -2,18 +2,18 @@
 // No theme
 
 @mixin button-base {
+	align-items: center;
 	cursor: pointer;
-	display: inline-block;
+	display: inline-flex;
```

Comment:
Should they not only be flexed for the `--icon` modifiers?

Reply:
Wanted to keep them all using flexbox so that it no longer needed text-align to do the same thing as justify-content (to keep it center by default).

Also reduced duplication of justify-content and align-items in both `--icon-right` and `--icon-left` as they can live in button-base mixin.

Reply:
👍